cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ClusterLoadParameters.java

```diff
@@ -31,6 +38,8 @@ public ClusterLoadParameters(HttpServletRequest request, KafkaCruiseControlConfi
   protected void initParameters() throws UnsupportedEncodingException {
     super.initParameters();
     _time = ParameterUtils.time(_request);
+    _startMs = ParameterUtils.startMs(_request);
+    _endMs = ParameterUtils.endMs(_request);
```

Comment:
Nit: Is it possible to make the retrieval of `_startMs` and `_endMs` in this class consistent with `PartitionLoadParameters`? I noticed that the interpretation logic has been moved to `GetBrokerStatsRunnable` class. If we do this refactoring, maybe we can also get rid of `ClusterLoadParameters#time()`, which causes potential confusion due to duality with `end()`?